sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/PhoneNumberAsyncClient.java

```diff
@@ -53,6 +60,7 @@
 @ServiceClient(builder = PhoneNumberClientBuilder.class, isAsync = true)
 public final class PhoneNumberAsyncClient {
     private final ClientLogger logger = new ClientLogger(PhoneNumberAsyncClient.class);
+    private static final Duration DEFAULT_POLL_INTERVAL = Duration.ofSeconds(60);
```

Comment:
This doesn't seen right, doesn't it mean that the poll happens every 60s so if the caller doesn't specify a poll duration, it will take this long for their method to complete? Could you check what is the typical default duration and do we need to have a default at all? Maybe this should be required?

Reply:
I made it a required parameter now so we shouldn't have this problem anymore 👍